modules/repository/branch.go

```diff
@@ -23,8 +26,9 @@ func GetBranch(repo *models.Repository, branch string) (*git.Branch, error) {
 }
 
 // GetBranches returns all the branches of a repository
-func GetBranches(repo *models.Repository) ([]*git.Branch, error) {
-	return git.GetBranchesByPath(repo.RepoPath())
+// if limit = 0 it will not limit
```

Comment:
GetBranches returns branches from the repository, skipping skip initial branches and returning at most limit branches, or all branches if limit is 0.